charts/alfresco-connector-ms365/values.yaml

```diff
@@ -0,0 +1,44 @@
+global:
+  alfrescoRegistryPullSecrets: quay-registry-secret
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
```

Comment:
I think it doesn't make sense to have a global strategy as the strategy really depends on the service it self.
Move it to root .Values